g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/VersionMatcher.java

```diff
@@ -27,7 +27,6 @@ public static String getMatchedVersion(final String version,
             final List<String> vList) throws L3APIException {
```

Comment:
i don't see which codes trigger  throwing  this exception, should remove 'throws L3APIException'?

Reply:
done